tests/testthat/out/tibble-all--300-20.txt

```diff
@@ -3,3 +3,13 @@
 1  1.00     1 T    
 2  2.50     2 F    
 3 NA       NA NA   
+  d     e     
+  <chr> <fctr>
+1 a     a     
+2 b     b     
+3 <NA>  <NA>  
+  f         
```

Comment:
Do we want to add a newline here? Maybe it looks ok when formatted?

Reply:
The frieze is shown in subtle style, maybe that's enough? We could underline too... I'm worried about wasting a whole line for empty space.

Reply:
Yeah, maybe should include a few of my ideas from the tibble issue (i.e. remove underline + make types a colour + italic) and then review.